src/store/new-request-module.ts

```diff
@@ -2360,14 +2360,6 @@ export class NewRequestModule extends VuexModule {
 
       if (response?.data) {
         this.setNrResponse(response.data)
-
-        const { dispatch } = this.context
```

Comment:
Let's get rid of this style of getting properties!

Reply:
I think this fails silently, and I think when the Vue instance restarts -- for whatever reason -- these additional properties added to context or `Vue.prototype` get cleared, which may also be a root cause of various problems.